masterlist.yaml

```diff
@@ -9547,6 +9547,95 @@ plugins:
+      - name: 'Kapotun ARMA LegendsOfAetherium.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA LunarGuard.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA Moonpath.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA NFF.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA PracticalPirate.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA TAWOBA.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA TBD Bikini.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA TBDUnique Treasures.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA Unique Uniques.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA USMP.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA USSEP.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA WetAndCold.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA Winterhold Restored.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
+      - name: 'Kapotun ARMA Wyrmstooth.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=) or (checksum("DWKaPoTunRace.esp", 0x672EB2D2) and not version("DWKaPoTunRace.esp", "2.0.0", <))'
```

Comment:
You could simplify this by making a separate regex plugin for these ARMA patches in the *3RD Party Patches* section & just adding the original mod as an incompatibility, like so:
```yaml
  - name: 'Kapotun ARMA .+\.esp'
    url:
     - link: ''
       name: ''
    inc:
      - name: 'DWKaPoTunRace.esp'
        condition: 'version("DWKaPoTunRace.esp", "2.0.0", >=)'
```
Filling in the URL of course (since I don't know it, I don't recall if you ever provided it). You can do this in the same PR since it's still relevant to the mod at hand.

Reply:
These are compatible with old versions up to 1.3.1, should I keep condition as is in this case, even if update is strongly advised?

Reply:
The condition will prevent the incompatibilities from being displayed for users of older versions, so that shouldn't be an issue if that's what you're asking.